src/Payments.sol

```diff
@@ -672,80 +725,15 @@ contract Payments is
         );
     }
 
-    function updateOperatorLockupTracking(
-        OperatorApproval storage approval,
-        uint256 oldLockup,
-        uint256 newLockup
-    ) internal {
-        if (newLockup < oldLockup) {
-            uint256 lockupDecrease = oldLockup - newLockup;
-            approval.lockupUsage -= lockupDecrease;
-            return;
-        }
-
-        // Handle lockup increase
-        uint256 lockupIncrease = newLockup - oldLockup;
-
-        // Verify against allowance
-        require(
-            approval.lockupUsage + lockupIncrease <= approval.lockupAllowance,
-            "exceeds operator lockup allowance"
-        );
-
-        // Update usage
-        approval.lockupUsage += lockupIncrease;
-    }
-
-    function validateAndModifyRateChangeApproval(
```

Comment:
Don't need these anymore as allowance tracking and updates now happen in the `settleAccountLockupBeforeAndAfterForRail` modifier.